tests/ui/manual_assert.edition2021.fixed

```diff
@@ -27,8 +28,8 @@ fn main() {
     {
         panic!("qaqaq{:?}", a);
     }
-    assert!(a.is_empty(), "qaqaq{:?}", a);
-    assert!(a.is_empty(), "qwqwq");
+    assert!(!(!a.is_empty()), "qaqaq{:?}", a);
+    assert!(!(!a.is_empty()), "qwqwq");
```

Comment:
Is this change in suggestion expected?

Reply:
It was expected yes. Ideally this should be handled by `Sugg` to avoid it but will eventually be simplified by the `non_minimal` bool. If it's a deal breaker can revert to the manual implementation in the lint

Reply:
Just wondering, what's the issue with the previous implementation?

Reply:
I just felt it should be a case common enough not to be handled manually by each lint which needs to negate an expression 😄

Reply:
That's fair, and I agree with it. But I think we don't need to rush to remove this from the lint at the moment 😄

We can leave this PR for the comment for now, and work on improving the above later (maybe move this feature to `Sugg`, etc etc.)

Reply:
Ok 👍 Actually the fix in `Sugg` wouldn't be trivial since it only store a string version of the snippet.